bwa.wdl

```diff
@@ -5,15 +5,14 @@ import "common.wdl" as common
 task Mem {
     input {
         String? preCommand
-        File inputR1
-        File? inputR2
+        FastqPair inputFastq
         BwaIndex bwaIndex
         String outputPath
         String? readgroup
 
         String? picardJar
 
-        Int threads = 1
+        Int threads = 2
```

Comment:
Should not the default be 1 in any case? This is just one job. I know that two programs are run, but I think it is a good default to let one job take one slot on the cluster. The cluster should limit the usage to 1 slot. We should not compensate for bad cluster implementations here.